CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for Crate Admin Interface
 Unreleased
 ==========
 
+ - Display ``license.ident`` in the Enterprise Edition.
```

Comment:
Are we making the changelog messages more user focused/friendly? If so, then I would go with something like 'License ident is now displayed in the Enterprise Edition".